rest/src/main/java/org/apache/unomi/rest/EventsCollectorEndpoint.java

```diff
@@ -154,11 +139,13 @@ private EventCollectorResponse doEvent(String eventsCollectorRequestAsString, Lo
             */
         } else {
             Profile sessionProfile = session.getProfile();
+            final String errorMessage = String.format("No valid profile found or persona found for profileId=%s, aborting request !", session.getProfileId());
             if (sessionProfile.getItemId() != null) {
                 // Reload up-to-date profile
                 profile = profileService.load(sessionProfile.getItemId());
                 if (profile == null || profile instanceof Persona) {
-                    throw new InternalServerErrorException(String.format("No valid profile found or persona found for profileId=%s, aborting request !", session.getProfileId()));
+                    logger.error(errorMessage);
```

Comment:
maybe protect this message with "if (logger.isDebugEnabled()) {"

Reply:
This one will be protected higher in the chain, meaning we can keep it as it is